storage/blackhole/ha_blackhole.h

```diff
@@ -20,7 +20,7 @@
 #include "thr_lock.h"                           /* THR_LOCK */
 #include "handler.h"                            /* handler */
 #include "table.h"                              /* TABLE_SHARE */
-
+#include "sql_const.h"
```

Comment:
```/* MAX_KEY */```

Reply:
I have fixed it. But, honestly, I don't understand this rule. Ok, now this header file is used here only for the only constant. But, in the future somebody can use something declared in this header without even understanding it. And so what? Must the comment be changed? Why?

Reply:
You are right, this is useful only if it survives possible future changes, which is hard, and better managed by tools like include-what-you-use. Let's consider this optional going forward.